src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/TestStartup.cs

```diff
@@ -30,9 +32,11 @@ public void ConfigureServices(IServiceCollection services)
                 options => { options.SerializerSettings.NullValueHandling = NullValueHandling.Ignore; }
             );
 
-        services.AddControllers(
-                options => { options.ModelBinderProviders.Insert(0, new SeparatedQueryModelBinderProvider(",")); }
-            )
+        services.AddControllers(options =>
+            {
+                options.AddCommaSeparatedQueryModelBinderProvider();
+                options.AddTrimStringBinderProvider();
```

Comment:
I was wondering whether we definitely want to trim spaces? I wonder if we want to instead error if there is spaces in query params?

Reply:
Model validation occurs after model binding.
If a required field bound to a query parameter is empty after the trimming done by the model binder, it will error.